src/lambdas/lambda_delete_file.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict
+from .common import *
+import base64
+
+def base64url_decode(input):
+    return base64.urlsafe_b64decode(input + '==')
+
+
+def manage_token(auth_header: str) -> str:
+    parts = auth_header.split()
+    jwt = parts[1]
+
+    jwt_body_str: str = jwt.split('.')[1]
+    jwt_body: Dict = json.loads(base64url_decode(jwt_body_str))
+    username = jwt_body['username']
+
+    return username
```

Comment:
Redundant, these two functions are already in `common.py`

Reply:
[b86e778](https://github.com/magley/klau-drive/pull/24/commits/b86e7782d7a7fa9e46ef482dca1f74d5b1fdb242)